### DISCUSSION TITLE
3 Questions on extensions

### DISCUSSION THREAD

**Discussion by gitihobo** [oobabooga/text-generation-webui#1482] [Q&A]
Updated thread. How can I turn extensions on by default? when clicking start.bat?

**Comment by BetaDoggo** [comment]
1. SD API is as safe are running the SD webui normally, nothing ever leaves your computer.

2. To make an extension load on startup add the `--extensions` argument followed by the names of the extensions that you want to load to your start.bat, there should be a spot for command-line args.

3. New models go in the models folder inside the webui folder. If you're using the auto installer version it may be inside an installer files directory.

**Reply by gitihobo** [reply]
I have only seen arguments be placed inside the startwindows.bat folder, call python webui.py --extensions silero_tts sd_api_pictures is how I should put it? Also, there is download-model.py file but it just closes every time I try to open it

**Reply by BetaDoggo** [reply]
Yes that syntax looks correct. 

I'm not sure why the download models script isn't working, maybe you're missing a dependency. 
Try running `pip install requests` then trying again. If you start the script from cmd it should give you the reason that it exits.

**Reply by gitihobo** [reply]
pip installed but all are satisfied, I did get a no "request" module present error when trying model download bat but also, the extensions werent recognized when inserted in call python webui.py
